README.md

```diff
@@ -18,15 +18,16 @@ Or install it yourself as:
 
 ## Usage
 
-    cobratest [OPTION] [application path]
```

Comment:
This change in indentation will make these lines no longer show as code, which is not in line with the style of the rest of the doc.